<ui version="4.0" >
 <class>Form</class>
 <widget class="QDialog" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Liczby pierwsze</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mywindow.qrc" >:/images/qt.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QListWidget" name="listWidget" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Pozostało wątków:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pushButtonThread" >
     <property name="text" >
      <string>Wątki</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pushButtonStd" >
     <property name="text" >
      <string>Standard</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mywindow.qrc" />
 </resources>
 <connections/>
</ui>
